### PR TITLE
Change version number from 2.4 to 2.3 - NuGet Test Utilities package

### PR DESCRIPTION
**Pull request by CDiaz-MS**
## Overview
Changes version number from 2.4 to 2.3 for the NuGet Test Utilities PR: #7032 

## Changes
- Updates version number

### CODE REVIEW

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -668,7 +668,9 @@ PlayerSettings:
   webGLThreadsSupport: 0
   scriptingDefineSymbols:
     1: 
-    14: 
+    4: ARFOUNDATION_PRESENT
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset]: We don't want to commit these defines, but let the code resolve them when the project is loaded in the editor.

Comment (keveleigh) [reply ProjectSettings/ProjectSettings.asset]: Otherwise, looks good!

Comment (CDiaz-MS) [reply ProjectSettings/ProjectSettings.asset]: Ok, they are removed.

**Pull request merged**